code/src/layouts/dashboard-tabs/DashboardTabs.constants.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode } from "react";
+
+import PeopleIcon from "@mui/icons-material/People";
+import ReceiptLongIcon from "@mui/icons-material/ReceiptLong";
+import StorefrontIcon from "@mui/icons-material/Storefront";
+
+import OrdersTab from "@/layouts/dashboard-tabs/components/orders-tab/OrdersTab";
+import ProductsTab from "@/layouts/dashboard-tabs/components/products-tab/ProductsTab";
+import UsersTab from "@/layouts/dashboard-tabs/components/users-tab/UsersTab";
+
+type DashboardTab = {
+  labelTranslationKey: string;
+  name: DashboardTabName;
+  icon: ReactNode;
+  content: ReactNode;
+};
+
+export const DASHBOARD_TAB_NAMES = {
+  USERS: "users",
+  ORDERS: "orders",
+  PRODUCTS: "products"
+} as const;
+
+export type DashboardTabName =
+  (typeof DASHBOARD_TAB_NAMES)[keyof typeof DASHBOARD_TAB_NAMES];
```

Comment:
maybe `keyof typeof DASHBOARD_TAB_NAMES` will be enough for this type ?

Reply:
No, because we need values of object. I think we can try to create reusable type, I will try to do it